model/infinispan/src/main/java/org/keycloak/connections/infinispan/DefaultInfinispanConnectionProviderFactory.java

```diff
@@ -120,7 +116,7 @@ public InfinispanConnectionProvider create(KeycloakSession session) {
 
         return InfinispanUtils.isRemoteInfinispan() ?
                 new RemoteInfinispanConnectionProvider(cacheManager, remoteCacheManager, topologyInfo) :
-                new DefaultInfinispanConnectionProvider(cacheManager, remoteCacheProvider, topologyInfo);
+                new DefaultInfinispanConnectionProvider(cacheManager, topologyInfo);
```

Comment:
We can remove `DefaultInfinispanConnectionProviderFactory#runWithReadLockOnCacheManager` as it's no longer used.